src/index.js

```diff
@@ -1,17 +1,48 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import { Provider } from 'react-redux';
+import { createStore } from 'redux';
+import { v4 as uuidv4 } from 'uuid';
+import App from './components/App';
+import bookReducers from './reducers/index';
+
```

Comment:
**[Optional]** You are not importing the bookReducer but the combineReducer. Try to use descriptive naming.